test/integration/master/transformation_testcase.go

```diff
@@ -30,7 +30,7 @@ import (
 	"k8s.io/klog"
 
 	"github.com/coreos/etcd/clientv3"
-	"github.com/prometheus/client_golang/prometheus"
+	"k8s.io/component-base/metrics/legacyregistry"
```

Comment:
This is not -1 against this PR at all, just a question:

The name "legacy" makes me unconfident because that seems deprecated component.
I tried to find that is deprecated or not on component-base side, but I cannot.

One more thing is the component-base contains KubeRegistry interface which also implements prometheus.Gatherer interfance like https://github.com/kubernetes/component-base/blob/e1c485d3541add9d20eaf26c84602e13d76f3421/metrics/registry.go#L60

Can we use it instead for stable interface?

Reply:
Thanks your review. maybe the name "legacy" looks like unconfident. you can see [example](https://github.com/kubernetes/kubernetes/pull/83610/files)

Reply:
@RainbowMango  can you   explain this to  oomichi ? Thanks !!!

Reply:
looks like there are more PRs coming in that also use the legacyregistry path.
perhaps @logicalhan and @sttts can confirm if this is correct.